operations/security_and_authentication/Add_LDAP_User_Federation.md

```diff
@@ -48,9 +50,11 @@ LDAP user federation is not currently configured in Keycloak. For example, if it
    
 2. Update the LDAP settings in the customizations.yaml file.
 
-   1. (Optional) Add the LDAP CA certificate in the certs.jks section of customizations.yaml.
+   1. (Optional) Add the LDAP CA certificate in the certs.jks.b64 section of customizations.yaml.
 
       Follow step 8 under the "Create Baseline System Customizations" header in the [Prepare Site Init](../../install/prepare_site_init.md) procedure.
+
+      Return to this procedure before running the command to inject and encrypt certs.jks.b64 into the customizations.yaml file in step 8.
```

Comment:
Do we want to skip the "inject and encrypt certs.jks.b64` step?  I see a re-encrypt step below but I don't see anything that encrypts the certs.jks.b64 that was created in step 8.

Reply:
I believe the reasoning was because of the incorrect path to the customizations.yaml file. I can try and come up with a way to correct that.

Reply:
We probably want to recreate that command in this doc then with the corrected paths.   Unless I'm missing something (which is possible), it looks to me like we create certs.jks.b64 in step 8 but then it never gets written into customizations.yaml if we skip that command.

Reply:
I believe I added the correct command. Please let me know if any changes are needed.